substrate/frame/election-provider-multi-phase/src/lib.rs

```diff
@@ -1353,6 +1355,35 @@ pub mod pallet {
 	pub struct Pallet<T>(_);
 }
 
+pub struct SnapshotWrapper<T>(PhantomData<T>);
```

Comment:
nit: Could add a small documentation that this wrapper handles the synchronization of `Snapshot`, `SnapshotMetadata` and `DesiredTargets`